docs/beast_development.rst

```diff
@@ -76,7 +76,7 @@ with:
 
 .. code-block:: console
 
-   $ python setup.py develop
+   $ tox -e develop

```

Comment:
Oops, will correct this. It seems that for this particular block I can just remove this line since it now doubles with the code just above on line 72: $ `pip install -e .`